packages/node_modules/@webex/plugin-meetings/src/roap/util.js

```diff
@@ -45,54 +43,16 @@ RoapUtil.ensureMeeting = (meeting, type) => {
   return true;
 };
 
-RoapUtil.compareLastRemoteOffer = (currentOffer, previousOffer) => {
-  let index1, index2;
-  let difference = false;
+RoapUtil.updatePeerConnection = (meeting, session) => PeerConnectionManager.updatePeerConnection({
```

Comment:
It would be great if we could start adding jsdoc to these functions. I know during initial development of the plugin we skipped it, but they would be really helpful for the core team who are not as familiar with the meetings plugin.